SpatialGDK/Documentation/content/generating-a-snapshot.md

```diff
@@ -4,7 +4,11 @@
 
 If you're not familiar with snapshots in the context of SpatialOS, please look at the [full snapshot documentation](https://docs.improbable.io/reference/latest/shared/operate/snapshots) (SpatialOS documentation).
 
-The SpatialOS GDK for Unreal snapshots contain two kinds of entities: critical entities and placeholders.
+The SpatialOS GDK for Unreal snapshots contain three kinds of entities: critical entities, stably named replicated actors and placeholders.
+
+### Stably named replicated actors
```

Comment:
Maybe we should start using the term Startup Actors?

Reply:
Agree with this. Easier to understand for someone new.